test/test_hl_api.py

```diff
@@ -12,16 +12,15 @@ def _objective_test_population(individual, rng_seed):
     if not individual.fitness_is_None():
         return individual
 
-    np.random.seed(rng_seed)
-
     n_function_evaluations = 100
 
     f_graph = individual.to_func()
 
     def f_target(x):  # target function
         return x[:, 0] - x[:, 1]
 
-    x = np.random.normal(size=(n_function_evaluations, 2))
+    rng = np.random.RandomState(seed=rng_seed)
```

Comment:
Why don't you use the fixture here?

Reply:
I think the intention here is to use the same random number stream in every call to the objective such that the fitness does *not* depend on the random number stream. I think if i'd use the fixture, then rngs are shared across objective calls, right? same reason below